From a856fe03089394072a3b4a3b6f466d631de318d4 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 20 Dec 2017 15:17:46 +0900
Subject: [PATCH 610/909] arm64: dts: r8a77965-salvator-x: Add MMNGRBUF driver
 support

commit d9120ecf533a0a0ab07f3b18802247641303140e from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds MMNGRBUF driver support for the R8A77965 SoC
on the Salvator-X board.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77965-salvator-x.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77965-salvator-x.dts b/arch/arm64/boot/dts/renesas/r8a77965-salvator-x.dts
index ca9e4c0cace7..05a16316ba3b 100644
--- a/arch/arm64/boot/dts/renesas/r8a77965-salvator-x.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77965-salvator-x.dts
@@ -57,6 +57,10 @@
 		compatible = "renesas,mmngr";
 		memory-region = <&mmp_reserved>, <&lossy_decompress>;
 	};
+
+	mmngrbuf {
+		compatible = "renesas,mmngrbuf";
+	};
 };
 
 &du {
-- 
2.17.1

